### PR TITLE
chore(vue): update to newer slot format in stories

### PR DESCRIPTION
**Pull request by Haroenv**
already updated in the components themselves

### CODE REVIEW

File: stories/utils.js [hunk 1]
```diff
@@ -38,7 +38,6 @@ export const previewWrapper = ({
       :search-client="searchClient"
       index-name="${indexName}"
       :routing="routing"
```
Comment (sarahdayan) [inline stories/utils.js:40]: Why remove Insights?

File: examples/media/src/App.vue [hunk 2]
```diff
@@ -100,8 +116,10 @@ import { history as historyRouter } from 'instantsearch.js/es/lib/routers';
 import { simple as simpleMapping } from 'instantsearch.js/es/lib/stateMappings';
 
 import './App.css';
+import __Template from '../../../src/components/__Template.vue';
 
 export default {
+  components: { __Template },
```
[the remark this answers is not in the document]
Comment (Haroenv) [reply examples/media/src/App.vue:122]: wrong import, thanks!

**Review by eunjae-lee — APPROVED** [review]

**Comment by eunjae-lee** [comment]
Did I approve it too early? Anyway, so far, it looks good :)

**Comment by Haroenv** [comment]
the examples were acting up, so I removed them from this commit and was waiting for the CI to pass to be sure @eunjae-lee, will make a new PR for the slots in the examples

**Pull request merged**